src/components/SearchResultsBox.jsx

```diff
@@ -1,34 +1,32 @@
 import React from 'react';
+import _ from 'lodash';
 import SearchResultsList from './SearchResultsList';
 import Loader from 'halogen/RingLoader';
 
 export default class SearchResultsBox extends React.Component {
-
-  getContent() {
+  render() {
     if(this.props.fetchingInProgress) {
       return (
         <center>
           <Loader color="#4BCEFF" size="140px" />
         </center>
       );
-    } else if(this.props.show) {
-      return (
-        <div>
-          <h4>Search Results for <span className="text-primary">{ this.props.searchTerm }</span></h4>
+    }
+
+    return (
+      <div>
+        <h4>Search Results for <span className="text-primary">   { this.props.searchTerm }</span></h4>
```

Comment:
Is there a reason that there are so many spaces between `<span className="text-primary">` and `{ this.props.searchTerm }`